druid-shell/src/platform/x11/window.rs

```diff
@@ -336,6 +350,29 @@ impl WindowBuilder {
             .context("set _NET_WM_PID")?;
         }
 
+        let size = (width_px as i32, height_px as i32);
+        let mut size_hints = WmSizeHints::new();
+        let min_size = self.min_size.to_px(scale);
+        if self.resizeable {
+            size_hints.min_size = Some((min_size.width as i32, min_size.height as i32));
+        } else {
+            size_hints.min_size = Some(size);
+            size_hints.max_size = Some(size);
+        }
+        log_x11!(size_hints
+            .set_normal_hints(conn.as_ref(), id)
+            .context("set wm normal hints"));
```

Comment:
This is similar to `Window::resizable` not sure how to de duplicate it

Reply:
Maybe have a `fn(size: Size, resizable: bool) -> WmSizeHints` somewhere?

Reply:
oh good idea